devenv/zabbix70/docker-compose.yml

```diff
@@ -0,0 +1,119 @@
+version: '3'
+
+services:
+  # Grafana
+  grafana:
+    image: grafana/grafana:10.1.2
+    ports:
+      - '3001:3000'
+    volumes:
+      - ../..:/grafana-zabbix
+      - ../dashboards:/devenv/dashboards
+      - ../grafana.ini:/etc/grafana/grafana.ini:ro
+      - '../datasources.yaml:/etc/grafana/provisioning/datasources/datasources.yaml'
+      - '../dashboards.yaml:/etc/grafana/provisioning/dashboards/dashboards.yaml'
+
+  # Zabbix
+  zabbix-server:
+    image: zabbix/zabbix-server-pgsql:alpine-trunk
```

Comment:
too bad they don't have a versioned one for `7`, like `zabbix/zabbix-server-pgsql:alpine-7.0-latest` 😿